vars/test.yml

```diff
@@ -99,6 +99,10 @@ esxi_password: "CHANGEME"
 
 validate_certs: false
 
+# NTP servers to be configured on vCenter Server and ESXi server for time synchronization
```

Comment:
Is this a test case specific parameter? then can put below in "Test cases parameters" part. And the name of this parameter, something like `vsphere_ntp_servers` is better to distinguish with the setting in VM.

Reply:
Thanks. All comments are addressed in https://github.com/vmware/ansible-vsphere-gos-validation/pull/722/commits/4bbedafce1935216b1d13e5f4e0f5a26375bc6e9.